server-mirror-git/src/test/java/com/linecorp/centraldogma/server/internal/mirror/DefaultMetaRepositoryWithMirrorTest.java

```diff
@@ -138,32 +138,28 @@ void testInvalidMirrors() {
     }
 
     @Test
-    void testSingleTypeMirror() {
+    void testMirror() {
         metaRepo.commit(Revision.HEAD, 0, Author.SYSTEM, "",
                         Change.ofJsonUpsert(
                                 PATH_MIRRORS,
                                 "[{" +
                                 "  \"enabled\": true," +
-                                "  \"type\": \"single\"," +
```

Comment:
Can we at least keep one entry with `"type:": "single"` to ensure the backward compatibility?

Reply:
It's here. 😉  https://github.com/line/centraldogma/pull/836/files#diff-42445cf029e7cfb1d5a9f00593163dd61061d62631ab68e3f8200e56a626870dR233-R235